src/main/java/uk/gov/hmcts/reform/hmc/service/AccessControlServiceImpl.java

```diff
@@ -99,7 +99,7 @@ public List<String> verifyCaseAccess(String caseReference, List<String> required
         return roleAssignmentsMatches.stream().map(RoleAssignment::getRoleName).collect(Collectors.toList());
     }
 
-    private List<RoleAssignment> verifyRoleAccess(List<String> requiredRoles) {
+    public List<RoleAssignment> verifyRoleAccess(List<String> requiredRoles) {
```

Comment:
Can we have a parent method like the overrides one so we can check on the param `if (!applicationParams.isAccessControlEnabled()) {
            return Collections.emptyList();
        }` then post this check call verifyRoleAccess